### DISCUSSION TITLE
Observable Inputs.table: how to add a caption?

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#3420] [Q&A] [answered]
Hi,
I'm using

~~~~
```{ojs}
//| echo: false
//| label: tbl-20162017
//| tbl-cap: "Penguin body mass by sex and species"
data_2016_2017 = FileAttachment("test.csv").csv({ typed: true })
Inputs.table(data_2016_2017)
```
~~~~

but I have

![image](https://user-images.githubusercontent.com/30607/203262637-9046c025-ae79-4240-b60c-0a029436c8dd.png)

What am I doing wrong?

Thank you

**Comment by cscheid** [comment, marked as answer]
Unfortunately, that's a current limitation, and you have to use "Figure" for OJS crossrefs.
